### PR TITLE
404 error page

### LINKED ISSUE #689: Implement 404 Error page
**Description**
Implement the 404 error page design discussed on #285

**Proposal**
Design: https://www.figma.com/file/m7obOydxAhGNmlFYvxTk3Z/GP-Explorer?node-id=1197%3A1531

### PR DESCRIPTION
**Pull request by alongoni**
# Summary

Closes #689

Redesign 404 error page.
After:
image
Before:
image

# To Test

1. Open the page:
https://pr701--gpui.review.gnosisdev.com/***/

### CODE REVIEW

**Comment by elena-zh** [comment]
LGTM!

**Comment by alongoni** [comment]
hey @alfetopito @anxolin I found that there are 3 more 404 error pages (gp-v1, safe-swap and trade).
image

Do we need to fix those as well?

**Comment by alfetopito** [comment]
> hey @alfetopito @anxolin I found that there are 3 more 404 error pages (gp-v1, safe-swap and trade). image
> 
> Do we need to fix those as well?

No need, those are all on-hold

**Review by alfetopito — APPROVED** [review]
LGTM too!

@biocom for CSS review

File: src/apps/explorer/pages/NotFound.tsx [hunk 1]
```diff
@@ -1,15 +1,68 @@
...
+  }
+`
+
+const Title = styled.h1`
+  margin: 3rem 0 2.95rem;
+  font-weight: ${({ theme }): string => theme.fontBold};
+`
+
+const Content = styled.div`
+  font-size: 16px;
```
Comment (biocom) [inline src/apps/explorer/pages/NotFound.tsx:27]: rem values

File: src/apps/explorer/pages/NotFound.tsx [hunk 2]
```diff
@@ -1,15 +1,68 @@
...
+
+  p {
+    line-height: ${({ theme }): string => theme.fontLineHeight};
+    overflow-wrap: break-word;
+  }
+`
+
+const StyledLink = styled(Link)`
+  height: 5rem;
+  border: 1px solid ${({ theme }): string => theme.borderPrimary};
```
Comment (biocom) [inline src/apps/explorer/pages/NotFound.tsx:44]: rem value ? ==> 0.1rem solid.........

**Review by biocom — APPROVED** [review]
Approved with comments (mostly convert px to rem).

**Review by henrypalacios — APPROVED** [review]
Nice Work!

**Review by alfetopito — APPROVED** [review]

**Pull request merged**